### PR TITLE
'apictl logout' revoke the currently used token when an user is logging out. [Fix for 327] [On 3.1.X Branch]

### PR DESCRIPTION
**Pull request by Chamindu36**
## Purpose
When a user tries to logout from an environment the generated access token for that user is not revoking and it keeps there. This has to be fixed as when the user is logging out that token has to be revoked. Therefore this has to be fixed as when an user logging out used token should be revoked.

**Reproducing the issue**

1. User login using apictl login
2. Do some work
3. apictl logout
4. Same user logins using "apictl login"
5. When doing next operations, we can see that the same previous token is used for those.

## Goals
Fixes issue No #327 for 3.1.x Branch

## Approach
**1. Create a function RevokeAccessToken()**
_This Function will create an HTTP post request to invoke token revocation endpoint and kill the current access token for that particular user before the user logs out from the environment._
**2. Use that method in logout.go**
_Before the user logout from the environment and erase credentials from .json file a newly introduced method will be executed to revoke the current access token._
**3. Provide tokenRevokeEndpoint.** 
_When providing this revoke token endpoint user does not need to do anything. Derivation of this revocation token endpoint will be done internally. That derived token revocation endpoint will be provided as the url RevokeAccessToken() function._

## User stories
Logging out from an environment in apictl.

## Documentation
No Document changes

## Test environment
Ubuntu 20.04
Java 1.8_252
APIM 3.1.0

### CODE REVIEW

File: import-export-cli/docs/apictl_set.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ apictl set --mode default
 ### Options
 
 ```
-      --export-directory string    Path to directory where APIs should be saved (default "/home/chamindu/.wso2apictl/exported")
+      --export-directory string    Path to directory where APIs should be saved (default "/home/chamindu/Desktop/Testing")
   -h, --help                       help for set
-      --http-request-timeout int   Timeout for HTTP Client (default 10000)
+      --http-request-timeout int   Timeout for HTTP Client (default 27823)
```
Comment (npamudika) [inline import-export-cli/docs/apictl_set.md:33]: why is this changed?

Comment (Chamindu36) [reply import-export-cli/docs/apictl_set.md:33]: I'll Change that @npamudika. Changed that for testing purposes.
